src/app/app-routing.module.ts

```diff
@@ -1,8 +1,13 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { AdminFormCreateEventComponent } from './components/admin-form-create-event/admin-form-create-event.component';
+import { AdminFormManageEventComponent } from './components/admin-form-manage-event/admin-form-manage-event.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'new/advertisingconfig', component: AdminFormCreateEventComponent },
```

Comment:
I think I've settled on a preference for: new/campaign, manage/campaign

I think that is clean and understandable

Reply:
gotcha, i've updated it. Since I was able to use the tabs, this path might be better named something more general since it includes both create and manage functionalities. path: 'campaigns'  Thoughts?

Reply:
Not a bad Idea.  I like the idea of having the 2 routes available for us.  In the future we can use them to decide which tab we start on when the component loads.  So, if a link form another part of the site goes to manage/campaign then it will go to the same component but start on the manage tab.